health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/hooks/services/createUpdatePlanProject.js

```
@@ -895,26 +909,99 @@ const createUpdatePlanProject = async (req) => {
-              message: "SETUP_COMPLETED",
+              message: "DRAFT_COMPLETED",
               back: "BACK_TO_HOME",
               backlink: `/${window.contextPath}/employee`,
-              responseId: planResForCompleteSetup?.PlanConfiguration?.[0]?.name,
-              info: "SETUP_MICROPLAN_SUCCESS_NAME",
+              responseId: fetchedPlanForSummary?.name,
+              info: "DRAFT_MICROPLAN_SUCCESS_NAME",
+              showDraftDownload:true,
+              actionLabel: "DOWNLOAD_DRAFT_MICROPLAN",
+              planObject : fetchedPlanForSummary
               // description: "SETUP_MICROPLAN_SUCCESS_RESPONSE_DESC",
```

Biome (1.9.4)

[error] This case is falling through to the next case.

Add a `break` or `return` statement to the end of this case to prevent fallthrough.

(lint/suspicious/noFallthroughSwitchClause)